src/core/adapters/recommendations.ts

```diff
@@ -66,8 +66,8 @@ namespace Recommendations {
   };
 
   // tslint:disable-next-line max-line-length
-  export const addLocationToRequest = (request, state, config: Configuration): RecommendationsBody | RecommendationsRequest => {
-    const locationConfig = ConfigurationAdapter.extractLocation(config);
+  export const addLocationToRequest = (request, state): RecommendationsBody | RecommendationsRequest => {
```

Comment:
I know they aren't there currently, but we should add the types of the parameters `request` and `state` here